src/main/groovy/de/triplet/gradle/play/PlayPublisherPluginExtension.groovy

```diff
@@ -24,4 +24,14 @@ class PlayPublisherPluginExtension {
         return track
     }
 
+    private Double userFraction = 0.1
```

Comment:
This could be a Groovy Bean if you remove the `private` modifier and the explicit getter and setter.